common/src/main/java/com/google/i18n/addressinput/common/FormOptions.java

```diff
@@ -125,16 +124,6 @@ public FormOptions setCustomFieldOrder(String regionCode, AddressField... fields
     return this;
   }
 
-  /**
-   * Blacklist the given CLDR (Common Locale Data Repository) region (country) code
-   * (http://goto/iii) indicating countries that for legal or other reasons should not be available.
-   * <p>
-   * Calls are cumulative, call this method once for each region that needs to be blacklisted.
-   * <p>
-   * We reserve the right to change this API from taking individual regions to taking a set. Before
-   * this can happen we need to decouple the iOS I18NFormOptions class from this class (j2objc) to
-   * ensure that Java vargs / collection args don't spill over to the iOS API.
-   */
```

Comment:
Hmm, we should get rid of most of this, but some of it is still good. Is this auto-removed?